reflex/config.py

```diff
@@ -223,24 +223,32 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        # set overriden class attribute values as os env variables to avoid losing them
+        # Load config values from env_path and override os_env if override_os_env is True
+        if self.env_path is not None and os.path.isfile(self.env_path):
+            load_dotenv(self.env_path, override=self.override_os_envs)  # type: ignore
+            # Recompute constants after loading env variables
+            importlib.reload(constants)
+            # Recompute instance attributes
+            self.recompute_field_values()
+
+        # Set class attributes not present in os_env in os_env
         for key, value in dict(self).items():
```

Comment:
i dont think this code needs to set values in the environment that were actually provided in the initializer of the config.

* if they come from the `env_path` file, then they will already be in the environment, so no need to set
* if they are set after `Config` initialization then they will need to be applied to the environment in `__setattr__`
* otherwise the values are provided when initializing `Config`, and subsequent `get_config` calls in a subprocess will import `rxconfig.py`

The only values that really need to be put into the environment are those set _after_ initialization.

Reply:
> i dont think this code needs to set values in the environment that were actually provided in the initializer of the config.
> 
> * if they come from the `env_path` file, then they will already be in the environment, so no need to set
> * if they are set after `Config` initialization then they will need to be applied to the environment in `__setattr__`
> * otherwise the values are provided when initializing `Config`, and subsequent `get_config` calls in a subprocess will import `rxconfig.py`
> 
> The only values that really need to be put into the environment are those set _after_ initialization.

oh! I did not quite think of it like this. I am surely going to attempt to correct this.